### PR TITLE
fetch user data from servlet to display or hide the comment form

### PR DESCRIPTION
**Pull request by clayferg**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/LoginAPI.java [hunk 1]
```diff
@@ -40,39 +40,42 @@
...
     UserService userService = UserServiceFactory.getUserService();
-    if (userService.isUserLoggedIn()) {
-      String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = "/index.html"; 
-      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut); 
-      response.getWriter().println("<p>You're logged in, good job.</p>");
-      response.getWriter().println("<p>Your email is: " + userEmail + "!</p>");
-      response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>"); 
+    boolean isLoggedIn = userService.isUserLoggedIn(); 
+    String loginLink;
```
Comment (ciddu) [inline portfolio/src/main/java/com/google/sps/servlets/LoginAPI.java:61]: You could use ternary operator https://www.baeldung.com/java-ternary-operator

String loginLink = (isLoggedIn): null : userService.createLoginURL("/index.html");
String logoutLink = (isLoggedIn): userService.createLoginURL("/index.html") : null;

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -40,6 +40,23 @@ function addFavoriteThing() {
     factContainer.innerText = 'That is all the info you get for now...';
   }
 }
+
+function getUserInfo() {
+  fetch('login-check').then((response) => response.json()).then((loginInfo) => {
+    if (loginInfo.isLoggedIn) {
+      console.log(loginInfo.logoutLink);
+      document.getElementById("comment-form").style.display = "block";
+      document.getElementById("login-logout").href = loginInfo.logoutLink;
```
Comment (ciddu) [inline portfolio/src/main/webapp/script.js:49]: this link would be same regardless of logout or login except for the inner text. do you want to move it outside and just call it a link

File: portfolio/src/main/java/com/google/sps/servlets/LoginAPI.java (hunk 1 above)
Comment (clayferg) [reply portfolio/src/main/java/com/google/sps/servlets/LoginAPI.java]: Cool, I had seen that in the javascript tutorials, but I didn't know it worked for java too.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (clayferg) [reply portfolio/src/main/webapp/script.js]: Right, didn't think about that. I moved it outside.

**Pull request merged**